tests/testsuite/cargo_metadata/help/mod.rs

```diff
@@ -12,3 +12,26 @@ fn case() {
         .stdout_matches(file!["stdout.term.svg"])
         .stderr_matches(str![""]);
 }
+
+#[cfg(windows)]
+#[cargo_test]
+fn windows_verbatim_disk_case() {
```

Comment:
Cargo makes a core assumption that verbatim paths aren't used. This is likely not the only thing broken when people use our API with them.

Reply:
Okay~ Hmmmm... Should it possible we update the used `glob` crate version that after it published a newer version? And I will remove this unnecessary test fn then.

Reply:
I found another root cause about this issue in `cargo-modules` crate.

Thanks for your help~~~